src/DurableTask.Core/Query/IOrchestrationServiceQueryClient.cs

```diff
@@ -0,0 +1,20 @@
+﻿
+namespace DurableTask.Core.Query
+{
+    using System.Threading.Tasks;
+    using System.Threading;
+
+    /// <summary>
+    /// Interface to allow query multi-instance status with filter.
+    /// </summary>
+    public interface IOrchestrationServiceQueryClient
+    {
+        /// <summary>
+        /// Gets the status of all orchestration instances with paging that match the specified conditions.
+        /// </summary>
+        /// <param name="condition">Return orchestration instances that match the specified conditions.</param>
+        /// <param name="cancellationToken">Cancellation token that can be used to cancel the status query operation.</param>
```

Comment:
Remove the word "status" here.
```suggestion
        /// <param name="cancellationToken">Cancellation token that can be used to cancel the query operation.</param>
```